sdk/storagecache/Azure.ResourceManager.StorageCache/api/Azure.ResourceManager.StorageCache.netstandard2.0.cs

```diff
@@ -1,101 +1,99 @@
+        public Azure.ResourceManager.StorageCache.Models.StorageCacheDirectorySettings DirectoryServicesSettings { get { throw null; } set { } }
+        public Azure.ResourceManager.StorageCache.Models.StorageCacheEncryptionSettings EncryptionSettings { get { throw null; } set { } }
+        public Azure.ResourceManager.StorageCache.Models.StorageCacheHealth Health { get { throw null; } }
         public Azure.ResourceManager.Models.ManagedServiceIdentity Identity { get { throw null; } set { } }
-        public System.Collections.Generic.IReadOnlyList<string> MountAddresses { get { throw null; } }
-        public Azure.ResourceManager.StorageCache.Models.CacheNetworkSettings NetworkSettings { get { throw null; } set { } }
+        public System.Collections.Generic.IReadOnlyList<System.Net.IPAddress> MountAddresses { get { throw null; } }
+        public Azure.ResourceManager.StorageCache.Models.StorageCacheNetworkSettings NetworkSettings { get { throw null; } set { } }
         public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.StorageCache.Models.PrimingJob> PrimingJobs { get { throw null; } }
-        public Azure.ResourceManager.StorageCache.Models.ProvisioningStateType? ProvisioningState { get { throw null; } }
+        public Azure.ResourceManager.StorageCache.Models.StorageCacheProvisioningStateType? ProvisioningState { get { throw null; } }
         public System.Collections.Generic.IList<Azure.ResourceManager.StorageCache.Models.NfsAccessPolicy> SecurityAccessPolicies { get { throw null; } }
         public string SkuName { get { throw null; } set { } }
         public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.StorageCache.Models.StorageTargetSpaceAllocation> SpaceAllocation { get { throw null; } }
-        public string Subnet { get { throw null; } set { } }
-        public Azure.ResourceManager.StorageCache.Models.CacheUpgradeSettings UpgradeSettings { get { throw null; } set { } }
-        public Azure.ResourceManager.StorageCache.Models.CacheUpgradeStatus UpgradeStatus { get { throw null; } }
+        public Azure.Core.ResourceIdentifier Subnet { get { throw null; } set { } }
+        public Azure.ResourceManager.StorageCache.Models.StorageCacheUpgradeSettings UpgradeSettings { get { throw null; } set { } }
+        public Azure.ResourceManager.StorageCache.Models.StorageCacheUpgradeStatus UpgradeStatus { get { throw null; } }
         public System.Collections.Generic.IList<string> Zones { get { throw null; } }
     }
-    public partial class CacheResource : Azure.ResourceManager.ArmResource
+    public static partial class StorageCacheExtensions
+    {
+        public static Azure.Pageable<Azure.ResourceManager.StorageCache.Models.StorageCacheUsage> GetAscUsages(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.Core.AzureLocation location, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Does the `Asc` here mean `Azure Storage Cache`? We might need to spell it out here